plugins/postcss-cascade-layers/src/compound-selector-order.ts

```diff
@@ -67,9 +67,6 @@ function selectorTypeOrder(selector, type) {
 		return selectorTypeOrderIndex.pseudoElement;
 	}
 
-	if (type === 'pseudo' && selector && selector.indexOf('::') === 0) {
```

Comment:
`selectorParser.isPseudoElement(selector)` on line 66 expects a `Node` and this didn't work before because it was a string.

This code on line 70 was insufficient but did work with strings.

Same change in all other plugins with similar logic.